<launch>
    <node pkg="path_integration_3x3" name="region_visualizer" type="region_visualizer.py">
    </node>
</launch>
